pkg/sql/drop_role.go

```diff
@@ -476,13 +506,13 @@ func addDependentPrivileges(
 			var err error
 			if role.ForAllRoles {
 				err = errors.Newf(
-					"privileges for default privileges on new %s for all roles",
-					objectType,
+					wrapErrorMsgWithSchemaName("privileges for default privileges on new %s for all roles in database %s"),
+					objectType, dbName,
 				)
 			} else {
 				err = errors.Newf(
-					"privileges for default privileges on new %s belonging to role %s",
-					objectType, role.Role,
+					wrapErrorMsgWithSchemaName("privileges for default privileges on new %s belonging to role %s in database %s"),
```

Comment:
any reason not to use `errors.WithDetail` here? should we add a `errors.WithHint` to suggest what syntax can be used to remove the default privileges?

Reply:
We can add a hint for each but I'm slightly afraid it might make it _too_ verbose, but I think we'd rather err that way than making it more obfuscated so I'm for it.

Reply:
Yeah alright but I'd say the table info should be a detail